src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj

```diff
@@ -27,46 +27,51 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <SupportedPlatform Include="browser" />
+        <SupportedPlatform Include="browser"/>
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.6" />
-        <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
+        <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.6"/>
+        <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1"/>
     </ItemGroup>
 
     <ItemGroup>
-      <None Include="ReadMe.md" Pack="true" PackagePath="" />
-      <None Include="dymaptic_logo.png" Pack="true" PackagePath="" />
-      <Content Remove="package.json" />
-      <Content Remove="package-lock.json" />
-      <Content Remove="tsconfig.json" />
-      <Content Include="package.json" Pack="false" />
-      <Content Include="package-lock.json" Pack="false" />
-      <Content Include="tsconfig.json" Pack="false" />
-        <None Include="Scripts/arcGisJsInterop.ts" Pack="true" PackagePack="Scripts" />
-        <None Include="Scripts/definitions.d.ts" Pack="true" PackagePack="Scripts" />
+        <None Include="dymaptic_logo.png" Pack="true" PackagePath=""/>
+        <Content Remove="package.json"/>
+        <Content Remove="package-lock.json"/>
+        <Content Remove="tsconfig.json"/>
+        <Content Include="package.json" Pack="false"/>
+        <Content Include="package-lock.json" Pack="false"/>
+        <Content Include="tsconfig.json" Pack="false"/>
+        <None Include="Scripts/arcGisJsInterop.ts" Pack="true" PackagePack="Scripts"/>
+        <None Include="Scripts/definitions.d.ts" Pack="true" PackagePack="Scripts"/>
+        <None Remove="ReadMe.md"/>
+        <None Include="ReadMe.md" Pack="true" PackagePath=""/>
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="wwwroot\js" />
+        <Folder Include="wwwroot\js"/>
     </ItemGroup>
-    
-    <Target Name="NPM Install" AfterTargets="PreBuildEvent">
-        <Exec Command="npm install" />
+
+    <Target Name="Copy Readme" AfterTargets="PreBuildEvent">
+        <Exec ContinueOnError="true" Command="robocopy ../../ ./ ReadMe.md /it"/>
+    </Target>
+
+    <Target Name="NPM Install" AfterTargets="Copy Readme">
```

Comment:
Looks like a reformatted file with a single change (NPM Intall?)

Reply:
It's actually the `Copy Readme` command added, and the NPM Install, which was already there, got shoved downward.